gov/src/lib.rs

```diff
@@ -0,0 +1,248 @@
+use std::ffi::OsString;
+use std::fs::read_to_string;
```

Comment:
I prefer not to import functions directly, as it takes extra effort to figure out on the callsite if it's a std function or a local function, so I usually do 

``````rust
use std::fs;
...
fs::read_to_string
```